src/bindings/python/fluxacct/accounting/bank_subcommands.py

```diff
@@ -176,6 +176,56 @@ def print_hierarchy(cur, bank, hierarchy_str, indent=""):
     return hierarchy_str
 
 
+def print_parsable_hierarchy(cur, bank, hierarchy_str, indent=""):
+    # look for all sub banks under this parent bank
+    select_stmt = "SELECT bank,shares,job_usage FROM bank_table WHERE parent_bank=?"
+    cur.execute(select_stmt, (bank,))
+    sub_banks = cur.fetchall()
+
+    if len(sub_banks) == 0:
+        # we've reached a bank with no sub banks, so print out every user
+        # under this bank
+        cur.execute(
+            "SELECT username,shares,job_usage,fairshare FROM association_table WHERE bank=?",
+            (bank,),
+        )
+        users = cur.fetchall()
+        if users:
+            for user in users:
+                hierarchy_str += (
+                    indent
+                    + " "
+                    + bank
+                    + "|"
+                    + str(user[0])
+                    + "|"
+                    + str(user[1])
+                    + "|"
+                    + str(user[2])
+                    + "|"
+                    + str(user[3])
+                    + "\n"
```

Comment:
did you consider `f"{indent} {bank}|{user[0]}|{user[1]}|{user[2]}|{user[3]}\n"`?

Reply:
I didn't even consider f-strings 🤦 thank you for pointing that out! I'm gonna switch to using these instead; much cleaner